src/screens/TripSettings/TripSettings.tsx

```diff
@@ -14,13 +35,326 @@ type TripItineraryScreenNavigationProp = CompositeNavigationProp<
+                                label={t("tripName")}
+                                value={values.tripName}
+                                onChangeText={handleChange("tripName")}
+                                disabled={settingsMode == Mode.VIEW}
+                                onBlur={handleBlur("tripName")}
+                                errorMessage={
+                                    errors.tripName && touched.tripName
+                                        ? errors.tripName
+                                        : undefined
+                                }
+                                errorStyle={styles.errorMessage}
+                                placeholder={placeholder.tripName}
+                            />
+                            <Input
+                                label={t("description")}
+                                value={values.description ? values.description : ""}
+                                onChangeText={handleChange("description")}
+                                disabled={settingsMode == Mode.VIEW}
+                                onBlur={handleBlur("description")}
+                                multiline={true}
+                                numberOfLines={4}
+                                style={styles.textArea}
+                                errorMessage={
+                                    errors.description && touched.description
+                                        ? errors.description
+                                        : undefined
+                                }
+                                errorStyle={styles.errorMessage}
+                                placeholder={placeholder.description}
+                            />
+                            {errorUpdate && (
+                                <Text style={styles.errorText}>{errorUpdate.message}</Text>
+                            )}
+                            {settingsMode === Mode.EDIT && (
+                                <Button
+                                    containerStyle={styles.buttonContainer}
+                                    buttonStyle={styles.submitButton}
+                                    title={t("screens.tripSettings.submitUpdate")}
+                                    titleStyle={{
+                                        color: "black",
+                                        fontSize: 25,
+                                    }}
+                                    onPress={() => handleSubmit()}
+                                    loading={loadingUpdate}
+                                />
+                            )}
+                        </>
+                    )}
+                </Formik>
+                <Text h3 style={{ marginBottom: 15 }}>
+                    {t("screens.tripSettings.members")}
+                </Text>
+                {data.trip.members.map((member, i) => (
+                    <ListItem key={i} bottomDivider>
```

Comment:
```suggestion
                {data.trip.members.map((member) => (
                    <ListItem key={member.user.id} bottomDivider>
```
According to the [react docs](https://reactjs.org/docs/lists-and-keys.html#keys) they advise you to use unique identifiers if you have them before using the index which if I'm right the user id is